Adding License Key Instructions to Common Config (#3038)
FusionAuth/fusionauth-site: astro/src/content/docs/_shared/_common-configuration.mdx

> * Add a user with a known, valid email address and secure password within your organization.

Reviewer: On line 1 of this file, please add

import Breadcrumb from 'src/components/Breadcrumb.astro';

And make sure this renders correctly.

Author: Added in there, looks normal on my end.

* Register that user with the FusionAuth application and the `admin` role.

## Activate License Key

> Activating the license key will allow access to all of the features inside FusionAuth. The key is found in your account portal and is activated in the <strong>Reactor</strong> tab. Production keys must be used for any FusionAuth instance that customers will be logging into, that is how we track MAUs. Non-Production may be used for any other instances such as Dev,QA, or Sandbox. 

Reviewer: ```suggestion
Activating the license key will allow access to all of the paid features inside FusionAuth. The key is found in your [account portal](https://account.fusionauth.io/account/plan/) and is activated in the <Breadcrumb>Reactor</Breadcrumb> tab. 

Production keys must be used for any FusionAuth instance that customers will be logging into, that is how we track MAUs. 

Non-production keys may be used for any other instances such as your developer's laptops, QA, or UAT. 
```

* Add a user with a known, valid email address and secure password within your organization.
* Register that user with the FusionAuth application and the `admin` role.

## Activate License Key

Activating the license key will allow access to all of the paid features inside FusionAuth. The key is found in your [account portal](https://account.fusionauth.io/account/plan/) and is activated in the <Breadcrumb>Reactor</Breadcrumb> tab. 

Production keys must be used for any FusionAuth instance that customers will be logging into, that is how we track MAUs. 

> Non-production keys may be used for any other instances such as your developer's laptops, QA, or UAT. 

Reviewer: ```suggestion
Non-production keys may be used for any other instances such as your developers' laptops, QA, or UAT. 
```

* Log in to the administrative user interface.

> * Navigate to <strong>Plan</strong> and copy the appropiate key

Reviewer: ```suggestion
* Navigate to <strong>Plan</strong> and copy the appropriate key.
```

> * Navigate to <strong>Plan</strong> and copy the appropiate key

Reviewer: ```suggestion
* Navigate to <Breadcrumb>Plan</Breadcrumb> and copy the appropiate key
```

Author: manually added

> * Access the FusionAuth instance and navigate to the <strong>Reactor</strong> 

Reviewer: ```suggestion
* Access the FusionAuth instance and navigate to the <strong>Reactor</strong>.
```

Reviewer: breadcrumb is for all navigation.

> * Paste that key into the License Key field and Activate it

Reviewer: ```suggestion
* Paste that key into the License Key field and Activate it.
```

> [Full instructions on how to activate your license ](/docs/get-started/core-concepts/licensing)

Reviewer: ```suggestion
[Full instructions on how to activate your license](/docs/get-started/core-concepts/licensing).
```

> 

Reviewer: ```suggestion
```
